### PR TITLE
Remove <Loader/> from app setup pages and team user availability view

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

- Removes loader from app setup pages
- Replace old loader with skeleton loader for team user availability times 
- Add missing translations to team user availability

 Loom Video: https://www.loom.com/share/35c396e4e8a441a9a74574fead01367e

**Environment**: Staging(main branch)

### CODE REVIEW

File: apps/web/pages/v2/apps/[slug]/setup.tsx [hunk 1]
```diff
@@ -12,11 +12,7 @@ export default function SetupInformation(props: InferGetStaticPropsType<typeof g
   const { status } = useSession();
 
   if (status === "loading") {
-    return (
-      <div className="absolute z-50 flex h-screen w-full items-center bg-gray-200">
-        <Loader />
```
Comment (PeerRich) [inline apps/web/pages/v2/apps/[slug]/setup.tsx:17]: NIT: import Loader from "@calcom/ui/Loader";

still being imported

Comment (CarinaWolli) [reply apps/web/pages/v2/apps/[slug]/setup.tsx:17]: Good catch! I removed the import 👍

**Review by PeerRich — APPROVED** [review]
great stuff!

**Review by zomars — APPROVED** [review]
LGTM

**Pull request merged**